src/main/java/edu/neu/coe/info6205/util/SortBenchmark.java

```diff
@@ -174,8 +174,8 @@ void benchmarkStringSorters(String[] words, int nWords) {
             }
 
         if (isConfigBenchmarkStringSorter("heapsort") && nRunsLinearithmic > 0) {
-            Helper<String> helper = HelperFactory.create("Heapsort", nWords, config);
-            try (SortWithHelper<String> sorter = new HeapSort<>(helper)) {
+//            Helper<String> helper = HelperFactory.create("Heapsort", nWords, config);
+            try (SortWithHelper<String> sorter = new HeapSort<>(nWords, config)) {
```

Comment:
```Java
try (SortWithHelper<String> sorter = new HeapSort<>(nWords, nRuns, config))
```

Reply:
Yes, I have observed that.
I have not made that change for 2 reasons:

1. Out of scope of this current issue thread
2. Same behavior is observed in Quick sort dual pivot, so both heapsort and quicksort need that fix.
<img width="1347" alt="image" src="https://github.com/user-attachments/assets/cbf8e3e9-aa8f-45d0-b58f-989b3caafaef">